Libplanet/Net/Swarm.cs

```diff
@@ -75,7 +76,7 @@ public partial class Swarm : ICollection<Peer>, IDisposable
         public Swarm(
             PrivateKey privateKey,
             TimeSpan dialTimeout,
-            IPAddress ipAddress = null,
+            string host = null,
```

Comment:
What if an invalid address is passed?  Does the constructor throw an exception, or merely `Swarm` crashes at unspecified moments?

Reply:
Neither. `Swarm` will advertise the address as its endpoint, and as a result will be isolated from the network.

Do we need to check connectability? (It's probably hard on the constructor, and it's possible in `StartAsync()`).

Reply:
Although it would good to have basic validations on the given `ipAddress`, at this point we necessarily need that IMHO.

Reply:
`host` can be an IP address, but it can also be any host name. so I think it is a bit tricky to check here statically.